command/agent_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"crypto/x509"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
```

Comment:
I removed the deprecated `ioutil` functions I saw in this file as part of this PR. While it's not really related, I couldn't help but clean them up while I was in here.

Reply:
If required, I'm happy to remove these from the PR, but the noise felt small enough, and clean-up is good.